draft_1_proposal_instructions.md

```diff
@@ -0,0 +1,143 @@
+In the future, the contents of the "SNS lifecycle" section in ./README.md would
+be replaced with the content in this file (minus this preface, ofc).
+
+**TODO**: Once README is migrated to 1-proposal, we'll also need to update the
+Apple silicon section to mention that yq is needed.
+
+_This section assumes that you have successfully completed the instructions in
+the [Bootstrapping a testing environment via Docker] section above._
+
+This section explains how to perform SNS decentralization in the aforementioned
+testing environment (i.e. Docker container running a local canister execution
+environment where an NNS has been deployed).
+
+The commands mentioned here are to be run from within the Docker container.
+
+'run_basic_scenario.sh` also performs these steps. Therefore, reading that
```

Comment:
"performs these steps" is wrong as `run_basic_scenario.sh` does not perform the steps that follow in this draft of the 1-proposal approach

Reply:
Yes, the current version of the script does not do the 1-proposal flow YET, but that is the goal state. This assumes that we'll eventually reach the goal state.

I have added a TODO here that mentions what you are saying about how this does not reflect the current version.